backendAS/seedComments.js

```diff
@@ -54,7 +54,7 @@ class ReadComments extends Readable {
     const songname = faker.lorem.sentence(2); // string
     const artistname = faker.name.findName(); // String
     const imgurl = Math.floor(Math.random() * 200) + 1; // string
-    const hashtag = faker.random.word();
+    const hashtag = faker.lorem.word();
```

Comment:
This is a comment for line 48: `this._logPercentComplete();` *if* your data generation is taking a long time, consider reducing the number of console logs (it appears as though you are running a log function every 35 records? I could be wrong).  console logs are extremely slow operations.  If this statement is not relevant, then disregard